pymatgen/entries/computed_entries.py

```diff
@@ -565,6 +570,36 @@ def as_dict(self) -> dict:
         )
         return return_dict
 
+    def __eq__(self, other: object) -> bool:
+        # NOTE Scaled duplicates i.e. physically equivalent materials
+        # are not equal unless normalized separately
+        if self is other:
+            return True
+
+        if isinstance(other, self.__class__):
+            if self.entry_id != other.entry_id:
+                return False
+            if self._energy != other._energy:
+                return False
+            if self._composition != other._composition:
+                return False
+            # NOTE this is not performant and should be preceeded
+            # if faster rigorous checks are available.
+            return self.as_dict() == other.as_dict()
```

Comment:
If you're happy to set this as a sufficient requirement for equality of `ComputedEntry` I will adopt this as it will be faster. 

I was testing for differences and then full equality using `as_dict()` as I felt uncomfortable returning that two things are equal when they could easily be different.

Reply:
Yeah, I think if the user provides an `entry_id` then it's up to them to make sure they don't use it twice.